source/Camera/Basic/C++Halcon/CaptureViaGenICam/CaptureViaGenICam.cpp

```diff
@@ -4,43 +4,35 @@
 #include <chrono>
 #include <iostream>
 
-
-void savePointCloud(const HalconCpp::HObjectModel3D &model, const std::string &filename)
+void savePointCloud(const HalconCpp::HObjectModel3D &model, const std::string &fileName)
 {
-	model.WriteObjectModel3d(HalconCpp::HString{ "ply" },
-		HalconCpp::HString{ filename.c_str() },
-		HalconCpp::HString{ "invert_normals" },
-		HalconCpp::HString{ "false" });
+	model.WriteObjectModel3d(HalconCpp::HString{"ply"},
+							 HalconCpp::HString{fileName.c_str()},
+							 HalconCpp::HString{"invert_normals"},
+							 HalconCpp::HString{"false"});
```

Comment:
Did you manually add this space or is this clang-tidy stuff?

Reply:
nope, this must be some clang-tidy stuff